src/App/App.js

```diff
@@ -16,10 +18,29 @@ function App() {
         containerStyleName={'App__container'}
         badge={cardsContext.cards?.length || 0}
       />
-      <div className={cl('App__container', 'App__container-background')}>
-        {/* Should have context with cards */}
-        <CardList />
-      </div>
+
+      <Switch>
+
+        <Route exact path="/">
+          <div className={cl('App__container', 'App__container-background')}>
+            {/* Should have context with cards */}
+            <CardList />
+          </div>
+        </Route>
+
+        <Route path="/auth">
+          <div className={cl('App__container', 'App__container-background')}>
+            <Auth/>
+          </div>
+        </Route>
+
+        <Route path="/">
```

Comment:
Can you please refactor this case adding the 404 path and append redirecting if route is not defined below?